data/functions/itemized.sql

```diff
@@ -15,3 +121,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
```

Comment:
I realized we are missing one more thing in this - renaming the indexes and constraints after the tables are renamed.  I've made another note below where this happened on the Python side for reference.

Other constraints, such as the `CHECK` constraint for the child tables and any primary key constraints, should also be taken into account to ensure that the exist if and when a repartition has to be run so we still maintain zero down-time and disruption in API availability and performance.